app.py

```diff
@@ -87,7 +88,7 @@ def get_status():
         play = out[0].split()[1]
         if play == 'playing':
             r['playing'] = True
-        elif play == 'stopped':
```

Comment:
This change breaks the displayed title when stopping the playback as `response.playing` is missing from the dictionary. Appending to the (not existing) variable `msg` inside the JavaScript code therefore results in an artist which starts with `undefined`.